bitwarden_license/src/Commercial.Core/AdminConsole/Providers/RemoveOrganizationFromProviderCommand.cs

```diff
@@ -67,27 +67,31 @@ public class RemoveOrganizationFromProviderCommand : IRemoveOrganizationFromProv
 
         await _organizationRepository.ReplaceAsync(organization);
 
-        var customerUpdateOptions = new CustomerUpdateOptions
+        if (!string.IsNullOrEmpty(organization.GatewayCustomerId) &&
+            !string.IsNullOrEmpty(organization.GatewaySubscriptionId))
         {
-            Coupon = string.Empty,
-            Email = organization.BillingEmail
-        };
+            var customerUpdateOptions = new CustomerUpdateOptions
+            {
```

Comment:
Can you please move this `if` block to a private method and name it something descriptive? Partly to avoid excessive nesting, partly to help the reader understand what it's doing.

Reply:
Sure, addressed here: https://github.com/bitwarden/server/pull/3999/commits/2dad72a5c3ae826afff592278639c26431081a8e

Reply:
Great documentation, thank you!